cql3/statements/select_statement.cc

```diff
@@ -18,13 +18,17 @@
 #include "cql3/statements/prune_materialized_view_statement.hh"
 #include "cql3/statements/strongly_consistent_select_statement.hh"
 
+#include "exceptions/exceptions.hh"
+#include "seastar/core/future.hh"
+#include "seastar/coroutine/exception.hh"
```

Comment:
Nit: we usually use angle brackets for seastar imports (`#include <seastar/core/future.h>`). This is a stylistic nitpick, though I vaguely recall it might have some practical meaning and other reviewers/maintainers would complain.